coralproject/talk
# [next] MongoDB Indexes

## What does this PR do?

- Adds MongoDB Indexes
- Fixes bug related to shared comment moderation counts

## How do I test this PR?

1. Start up the server via `npm run start:development` or `npm run start`
2. Wait for the server to start serving HTTP requests (hit http://localhost:3000)
3. Using a MongoDB console, run `db.comments.getIndexes()` and see the indexes listed
